Makefile

```diff
@@ -20,14 +24,34 @@ mongo:
 	# else you should do mongod
 
 .PHONY: install
-install:
+install: ## Runs the install script and vendors golang dependencies
 	./scripts/install.sh
 	$(MAKE) vendor
 
-.PHONY: client
-client:
+.PHONY: goclient
+goclient: ## Runs the old golang survey client to play cribbage
 	go run localclient/main/main.go
 
+.PHONY: reactdev
+reactdev: ## Sets up the react client in dev mode
+	cd client/ && npm run dev
```

Comment:
Thanks - kept meaning to do this 😄

Reply:
Except it didn't work:
```bash
panic: html/template: pattern matches no files: `templates/*`
```
I think it's because running the go server from within `client` causes issues. So we need to change the `dev` script in `package.json`:
```json
  "scripts": {
    "build": "react-scripts build",
    "client": "react-scripts start",
    "server": "cd .. && make serve",
    "dev": "concurrently \"npm run server\" \"npm run client\""
  },
```